.github/workflows/integrationTest.yml

```diff
@@ -102,10 +102,12 @@ jobs:
           aws s3 cp s3://cloudwatch-agent-integration-bucket/integration-test/binary/${{ github.sha }} . --recursive
 
       - uses: montudor/action-zip@v1
+        if: steps.cached_win_zip.outputs.cache-hit != 'true'
```

Comment:
Ah this change was in here also. I could have sworn I saw code like what was published in #410